packages/locales/en/kafka.json

```diff
@@ -244,5 +244,13 @@
     "topic": "Topic"
   },
   "will_expire": "This trial instance will expire in <time>{{time}}</time>",
-  "will_expire_short": "Expires in <time>{{time}}</time>"
+  "will_expire_short": "Expires in <time>{{time}}</time>",
```

Comment:
I'd probably have added these strings to a new json file, but let's keep it this way for the moment, I still have to decide if the whole i18n makes sense the way is currently setup or if we should do something else.